### PR TITLE
Chore: GraphQL organised by features

### LINKED ISSUE #328: Refactor: GraphQL Domain Driven design of folder structure
Refactor folder structure to Domain Driven.
kiwi.wiki https://kiwi.wiki/incubator/universe/docs/graphql/directory-structure/
Check discussion https://skypicker.slack.com/archives/CEFT2NAE8/p1550248090020400

example: 
```
src
-------/booking
-------------/dataloaders
-------------/types
-------/locations
-------------/dataloaders
-------------/types
```

### PR DESCRIPTION
**Pull request by RobinCsl**
Summary: To follow best practices as explained in the `incubator/universe` documentation

Closes #328

### CODE REVIEW

File: apps/graphql/src/services/testingTools/TestingTools.js [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 import { graphql as originalGraphQL } from 'graphql';
 
-import schema from '../Schema';
-import createContext from './GraphQLContext';
+import schema from '../../Schema';
+import createContext from '../graphqlContext/GraphQLContext';
 
-jest.mock('./Fetch.js');
+jest.mock('../fetch/Fetch.js');
```
Comment (tbergq) [inline apps/graphql/src/services/testingTools/TestingTools.js]: Can you try and remove this mock and see if tests still work? We already mock fetch in `setupJest.js` (I know it is not related to this PR, but anyways 👼 )

Comment (RobinCsl) [reply apps/graphql/src/services/testingTools/TestingTools.js]: Sure, I'll try that!

Comment (RobinCsl) [reply apps/graphql/src/services/testingTools/TestingTools.js]: Good catch, it works well without this line.

**Comment by RobinCsl** [comment]
Waiting on #339  and #341 to be merged before fixing conflicts.

**Comment by RobinCsl** [comment]
@tbergq @FilipMessa 🚀

**Comment by michalsanger** [comment]
Just a small naming correction - it's called "feature based", the prev one was "role based".

**Pull request merged**